### ISSUE TITLE
We should consider if using restyled.io for syntax formatting is the right choice

### ISSUE THREAD

**Issue by mithro** [ryancj14/common-config#3]
We should consider if we want to adopt https://restyled.io/ for the code formatting stuff, it seems to have a lot of users and potentially does a lot of automation for us...

See also https://github.com/google/verible/issues/854

**Comment by ryancj14** [comment]
Xan is looking into restyled.io and will be implementing it here with formatter files in the aptly named `formatter-files` folder. After he has done a little more, maybe he can comment here with his thoughts. So far restyled is looking like a great option to implement.

**Comment by xanjohns** [comment]
@mithro I think restyled would be a great option. I have also made a [pull request](https://github.com/restyled-io/restylers/pull/219) to the restyled repository to add verible support, but it is still being approved. 

I believe we would need a Symbiflow organization admin to install [restyled.io](https://github.com/marketplace/restyled-io) from GitHub Marketplace, but we should probably wait until we have merged this repo as a subtree with other Symbiflow repositories, to ensure that we have the proper formatting configuration.
